serverless/telegram-handler/src/credentials.ts

```diff
@@ -58,3 +58,13 @@ export const getBotTokenFromId = async (botId: string): Promise<string> => {
 
   return botToken
 }
+
+/**
+ * Compares two bot tokens and throws an error if they do not match.
+ */
+export const verifyBotToken = (token1: string, token2: string): void => {
+  if (token1 !== token2) {
```

Comment:
Should probably use a [timing safe comparison method](https://nodejs.org/api/crypto.html#crypto_crypto_timingsafeequal_a_b) to check equality of bot token. [Reference on timing attacks](https://codahale.com/a-lesson-in-timing-attacks/).

Reply:
Learned something new, thanks for pointing this out!